core/src/main/scala/quasar/physical/ts/TSDestination.scala

```diff
@@ -166,7 +168,7 @@ final class TSDestination[F[_]: Concurrent: ContextShift: MonadResourceErr] priv
     |   ${config.schema.map(s => s"""--target_schema "$s"""").getOrElse("")}
     |   --target_table '$tableName'
     |   --field_separator ','
-    |   --null_value ''
+    |   --null_value '$NullSentinel'
```

Comment:
Why do we use a sentinel rather than disallow null values entirely?

Reply:
"Null" in this context just means `undefined`.

Reply:
ah, ok